### ISSUE TITLE
Retire phantom-scrape.es6.js?

### ISSUE THREAD

**Issue by foolip** [mdittmer/web-apis#44]
I noticed because it adds a dependency on PhantomJS which took a while to download. I think this was needed for ReSpec originally, but now we only use the static markup. Can it be removed, together with any extra dependencies?

**Comment by mdittmer** [comment]
Yes, it appears to be orphaned. In terms of other unneeded dependencies, it would be nice to get rid of them but I don't see it as a priority.
